### PR TITLE
Fixes #14

### LINKED ISSUE #14: Improve UI for creating and joining event
They should look more like the event modal

### PR DESCRIPTION
**Pull request by peterampazzo**

### CODE REVIEW

**Comment by peterampazzo** [comment]
@andtos90 Does this look good to you? Can I proceed applying these changes to `JoinEvent` component?

**Comment by peterampazzo** [comment]
This is how the new modal looks like 😄 

Screenshot 2020-04-07 at 16 26 53

File: src/containers/CreateEvent/CreateEvent.css [hunk 1]
```diff
@@ -4,3 +4,24 @@
...
+  top: 0px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  width: 350px;
+  height: 90%;
+  margin: auto;
+  border-radius: 15px;
+  overflow-y: auto;
+  -webkit-box-shadow: 3px 3px 5px 4px #ccc;  /* Safari 3-4, iOS 4.0.2 - 4.2, Android 2.3+ */
```
Comment (andtos90) [inline src/containers/CreateEvent/CreateEvent.css:24]: we don't need `-webkit-box` prefix, use only `box-shadow`

File: src/containers/JoinEvent/JoinEvent.css [hunk 2]
```diff
@@ -4,3 +4,24 @@
...
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  width: 350px;
+  height: 90%;
+  margin: auto;
+  border-radius: 15px;
+  overflow-y: auto;
+  -webkit-box-shadow: 3px 3px 5px 4px #ccc;  /* Safari 3-4, iOS 4.0.2 - 4.2, Android 2.3+ */
+  -moz-box-shadow:    3px 3px 5px 4px #ccc;  /* Firefox 3.5 - 3.6 */
```
Comment (andtos90) [inline src/containers/JoinEvent/JoinEvent.css:25]: we don't need `-moz-box` prefix, use only `box-shadow`

File: src/containers/JoinEvent/JoinEvent.css [hunk 3]
```diff
@@ -4,3 +4,24 @@
...
+  top: 0px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  width: 350px;
+  height: 90%;
+  margin: auto;
+  border-radius: 15px;
+  overflow-y: auto;
+  -webkit-box-shadow: 3px 3px 5px 4px #ccc;  /* Safari 3-4, iOS 4.0.2 - 4.2, Android 2.3+ */
```
Comment (andtos90) [inline src/containers/JoinEvent/JoinEvent.css:24]: we don't need `-webkit-box` prefix, use only `box-shadow`

File: src/containers/CreateEvent/CreateEvent.css [hunk 4]
```diff
@@ -4,3 +4,24 @@
...
+  -moz-box-shadow:    3px 3px 5px 4px #ccc;  /* Firefox 3.5 - 3.6 */
```
Comment (andtos90) [inline src/containers/CreateEvent/CreateEvent.css:25]: we don't need `-moz-box` prefix, use only `box-shadow`

**Pull request merged**